mlflow/entities/trace_location.py

```diff
@@ -178,11 +173,8 @@ def to_proto(self):
                 type=self.type.to_proto(),
                 inference_table=self.inference_table.to_proto(),
             )
-        elif self.uc_schema:
-            return pb.TraceLocation(
-                type=self.type.to_proto(),
-                uc_schema=self.uc_schema.to_proto(),
-            )
+        # uc schema is not supported in to_proto since it's databricks specific, should use
+        # databricks_service_utils to convert to proto
```

Comment:
Inference table is also databricks-specific; should it be moved out as well?

Reply:
Yes we can, but since it's already in the existing TraceLocation proto in service.proto, we shouldn't remove it right?